pkg/build/rpm_test.go

```diff
@@ -0,0 +1,54 @@
+package build
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/suse-edge/edge-image-builder/pkg/config"
+)
+
+func TestGetRPMFileNames(t *testing.T) {
+	// Setup
+	bc := config.BuildConfig{
+		ImageConfigDir: "../config/testdata",
```

Comment:
Since we're dealing with empty files, we can opt for creating temporary files in order to keep the repository clean. Please take a look at the `os.CreateTemp()` function.

Reply:
My thinking at the time was keeping the dir as a dir users could reference as an example to what the structure of config dir may look like. Let me know your thoughts